Docs/Architecture/KitbotExampleWalkthrough.md

```diff
@@ -143,7 +143,7 @@ To do this we need to make a Command factory method, or a method that returns `C
 
 ```Java
 public CommandBase setVoltagesCommand(DoubleSupplier left, DoubleSupplier right) {
-    return new RunCommand(() -> this.setVoltages(left.getAsDouble(), right.getAsDouble()), this);
+    return new RunCommand(this.setVoltages(left.getAsDouble(), right::getAsDouble), this);
```

Comment:
This one probably still needs the lambda and to revert the `right.getAsDouble()` call.